usd-rs/src/pxr/usd/prim.rs

```diff
@@ -78,6 +109,37 @@ impl Prim {
         }
     }
 
+    // this was causing segfault, hence the dirty implementation below.
+    /*     pub fn get_attributes(&self) -> Vec<Attribute> {
+        unsafe {
+            cpp!([self as "pxr::UsdPrim*"]
+                        -> Vec<Attribute> as "std::vector<pxr::UsdAttribute>" {
+                return self->GetAttributes();
+            })
+        }
+    } */
+
+    pub fn get_attributes(&self) -> Vec<Attribute> {
```

Comment:
Need to Wrap std::vector<UsdAttribute> so that we can return an object like this directly.